File: src/androidTest/java/burrows/apps/example/gif/TestApp.java
package burrows.apps.example.gif;

import burrows.apps.example.gif.di.component.AppComponent;
import burrows.apps.example.gif.di.component.ActivityComponent;

/**
 * @author <a href="mailto:jaredsburrows@gmail.com">Jared Burrows</a>
 */
public class TestApp extends App {
  public void setAppComponent(AppComponent appComponent) {
    this.appComponent = appComponent;
  }

  public void setRiffsyComponent(ActivityComponent netComponent) {
    this.activityComponent = netComponent;
  }
}

File: src/androidTest/kotlin/burrows/apps/example/gif/TestApp.kt
package burrows.apps.example.gif

import burrows.apps.example.gif.presentation.di.component.ActivityComponent

/**
 * @author [Jared Burrows](mailto:jaredsburrows@gmail.com)
 */
class TestApp : App() {
  fun setRiffsyComponent(netComponent: ActivityComponent) {
    this.activityComponent = netComponent
  }
}
